R/pairwise-comparisons.R

```diff
@@ -338,7 +374,9 @@ compare_two_models <- function(scores,
   forecast_unit <- get_forecast_unit(scores)
 
   if (!("model" %in% names(scores))) {
-    stop("pairwise comparisons require a column called 'model'")
+    cli_abort(
+      "pairwise comparisons require a column called 'model'"
```

Comment:
```suggestion
      "pairwise comparisons require a column called {.val {"model"}}."
```
would that work?

Reply:
I think we want to use `.var` here for variable?